src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
OK I don't that aspect, as we bypass the many optimizations the browser does to speed up site loading, but let's go with it and see what users think.